.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
+
+install: mvn -f gson install -DskipTests=true
+script: mvn -f gson test
```

Comment:
True. However, other sub-projects are in states of disrepair and not maintained well. We didn't want to tie them in with Gson.